### PR TITLE
Update data-protection-and-privacy-80e76fd.md

### PR DESCRIPTION
**Pull request by WolfgangTheilmann**
Refined DPP guidance + clarification of cross-consumed service scenarios.

### CODE REVIEW

**Review by juergen-walter — APPROVED** [review]

File: docs/data-protection-and-privacy-80e76fd.md [hunk 1]
```diff
@@ -9,6 +9,13 @@ For general information about data protection and privacy on SAP Business Techno
...
 
+> ### Note:  
+> SAP Cloud Logging is not designed to support the collection, processing, and storage of personal or business data.  
+We ask you take appropriate measures to ensure that no personal or business data is sent to our service.
+
+> ### Note:  
+> Cloud Logging service is storing all data taking into consideration specific data localization requirements (e.g. EU Access) of the region of your service consumption. The actual physical data center for storing the data might be different than the one where you consume the service, although the data center is within the same region.
```
Comment (keyes-bill) [inline docs/data-protection-and-privacy-80e76fd.md:17]: SAP Cloud Logging service stores data, taking into consideration specific data localization requirements (such as EU Access) of the region of your service consumption. The actual physical data center which stores the data might be different than the one where the service is consumed, although the data center is within the same region.

File: docs/data-protection-and-privacy-80e76fd.md [hunk 2]
```diff
@@ -9,6 +9,13 @@ For general information about data protection and privacy on SAP Business Techno
 > ### Note:  
 > SAP does not provide legal advice in any form. SAP software supports data protection compliance by providing security features and data protection-relevant functions, such as blocking and deletion of personal data. In many cases, compliance with applicable data protection and privacy laws is not covered by a product feature. Furthermore, this information should not be taken as advice or a recommendation regarding additional features that would be required in specific IT environments. Decisions related to data protection must be made on a case-by-case basis, taking into consideration the given system landscape and the applicable legal requirements. Definitions and other terms used in this documentation are not taken from a specific legal source. Handle personal data with care. You as the data controller are legally responsible when processing personal data.
 
+> ### Note:  
+> SAP Cloud Logging is not designed to support the collection, processing, and storage of personal or business data.  
+We ask you take appropriate measures to ensure that no personal or business data is sent to our service.
```
Comment (keyes-bill) [inline docs/data-protection-and-privacy-80e76fd.md:14]: Take appropriate measures to ensure that no personal or business data is sent to the service.

**Comment by keyes-bill** [comment]
I've made the changes in DITA. It should show up in the DRAFT help.sap by tomorrow.

**Comment by juergen-walter** [comment]
@keyes-bill please bring the changes to production.

I guess your changes will be propagated back automatically from DITA to this co-doc repository

**Comment by keyes-bill** [comment]
This has been pushed out to PRODUCTION. It should show up after the next build, which usually happens on Wednesdays or Thursdays (September 4-5).

**Comment by juergen-walter** [comment]
It is updated https://help.sap.com/docs/cloud-logging/sap-cloud-logging/data-protection-and-privacy

**Pull request merged**